src/cli.ts

```diff
@@ -106,9 +107,17 @@ getProjectRoot(pathArg || './src').then(async (projectRoot) => {
             })
 
             for (const group in result.reportOutput) {
-              log(`\n- <text_info> ${group}</text_info>`)
+              let groupOutput = group
+              if (argv.group === 'file') {
+                groupOutput = `\u001B]8;;file://${path.resolve(group)}\u001B\\${group}\u001B]8;;\u001B\\`
```

Comment:
What are these escape characters? Can you move them to the constants? They look scary 😱

Reply:
Sure, and they are a control sequence used by terminal emulators to create clickable hyperlinks.

reference:
https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda